server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestForceMergeAction.java

```diff
@@ -47,6 +47,9 @@ public String getName() {
 
     @Override
     public RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
+        if (request.hasContent()) {
+            throw new IllegalArgumentException("forcemerge takes arguments in http params, not as http body");
```

Comment:
These are "query parameters" (there are many different kinds of "parameters" in the HTTP specification, not only query parameters). I think we should say "request body" instead of "http body" (and let us capitalize "HTTP" if we keep it in the message.